README.md

```diff
@@ -93,6 +93,8 @@ about P4Info objects.
 P4Runtime `Entity` fields), along with `multicast_group_entry` and
 `clone_session_entry`, for runtime entity programming.
 
+`packet` is the command for packet IO, see the [usage](usage/packet_io.md) for more information.
```

Comment:
needs to be updated now that we have different commands for `packet_in` / `packet_out`